src/components/tooltip/tooltip.js

```diff
@@ -104,8 +104,9 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $interpolate,
 
     function addAriaLabel(override) {
```

Comment:
@jelbourn  - Per the PR ancestor (that was closed), we should add comments about *what* is `override` ?